src/math/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2019-2020, Lawrence Livermore National Security, LLC and
+# other Serac Project Developers. See the top-level LICENSE file for
+# details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+
+set(math_headers
+    expr_template_internal.hpp
+    expr_template_ops.hpp
+    mesh_utils.hpp
```

Comment:
Is mesh_utils really "math"?

Reply:
Hmmm. Maybe `numerics` or `discretization_utils` or something? I think these are all helper functions for the finite element operations.

Reply:
I think this should probably be renamed, but I can't think of a better one right now. I say we merge this and think about it.